backend/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+service nginx start
+uwsgi --http 127.0.0.1:5000 --wsgi-file app.py --callable app --need-app --die-on-term --uid www-data
```

Comment:
Do we want to create a systemd service for this in the future?

Reply:
probably the questions is how well systemd services in docker work